src/index.html

```diff
@@ -6,9 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="src/images" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="Moyo"
+        class="logo-img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#apple" class="nav-link is-active">Apple</a>
+          </li>
```

Comment:
Let's fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
```

Reply:
@volodymyr-soltys97  Expected class selector ".nav__list" to be kebab-case - спричиняє лінт ерори тому й змінила __ на -